src/GoogleAnalytics.js

```diff
@@ -0,0 +1,252 @@
+import { GoogleTagManager } from './GoogleTagManager';
+import { GoogleAnalyticsSettings } from './GoogleAnalyticsSettings';
+import { GoogleAnalyticsBridge } from './NativeBridges';
+
+// Exported Google analytics native bridged method that
+// accepts trackerId as first argument.
+const {
+  trackScreenView,
+  trackEvent,
+  trackScreenViewWithCustomDimensionValues,
+  trackEventWithCustomDimensionValues,
+  trackTiming,
+  trackPurchaseEvent,
+  trackMultiProductsPurchaseEvent,
+  trackException,
+  setUser,
+  allowIDFA,
+  trackSocialInteraction,
+  setTrackUncaughtExceptions,
+  setAppName,
+  setAppVersion,
+  setAnonymizeIp,
+} = GoogleAnalyticsBridge;
+
+export class GoogleAnalytics {
+  // Backwards compatibility
+  // GoogleTagManager was static property of GoogleAnalytics class in previous version.
+  static GoogleTagManager = GoogleTagManager;
+
+  constructor(trackerIds = []) {
+    if (Array.isArray(trackerIds)) {
+      this.setTrackerIds(trackerIds);
+    } else if (trackerIds) {
+      this.setTrackerId([trackerIds]);
+    } else {
+      this.setTrackerIds([]);
+    }
+  }
+
+  /**
+   * Backwards compatibility, old API support.
+   * @param {String} trackerId
+   */
+  setTrackerId(trackerId) {
+    this.setTrackerIds([trackerId]);
+  }
+
+  /**
+   * @param {Array} trackerIds
+   */
+  setTrackerIds(trackerIds) {
+    this.trackerIds = new Set(trackerIds);
+  }
+
+  /**
+   * Push trackerId into tracker ids set.
+   * @param {String} trackerId
+   */
+  addTrackerId(trackerId) {
+    this.trackerIds.push(trackerId);
```

Comment:
It will not work, this is error, worked with arrays before so this push left from old code. Fixed.